### PR TITLE
Skip button for Intro2

### PR DESCRIPTION
**Pull request by timobaehr**
skipbuttonintrotour-landscape
skipbuttonintrotour

### CODE REVIEW

**Comment by Andrew-Quebe** [comment]
Upon reviewing your code, it seems that a large amount of code has been removed from the `AppIntro` class. Any particular reason why?

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java [hunk 1]
```diff
@@ -17,78 +12,11 @@
 public abstract class AppIntro extends AppIntroBase {
     private static final String TAG = "AppIntro";
 
-    protected boolean skipButtonEnabled = true;
```
Comment (Andrew-Quebe) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java]: starting here, why was all this removed?

Comment (timobaehr) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java]: Hi, short answer is "moved" not "removed". 

In the current master the skip button is only used inside the ```AppIntro``` but not in ```AppIntro2```. I moved the skip button code from ```AppIntro``` into the ```AppIntroBase``` class as the skip button could be part of both intro tour variants.

Comment (Andrew-Quebe) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java]: Ah, very well.

**Pull request merged**